Makefile

```diff
@@ -50,7 +50,11 @@ docs: $(wildcard $(CRD_DEF)/*.go) ## Generate documentation
 	@go run -modfile tools/go.mod github.com/elastic/crd-ref-docs --config=docs/config.yaml --source-path=$(CRD_DEF) --templates-dir=docs/templates --output-path=docs/modules/ROOT/pages/reference.adoc
 	@go run ./docs
 
-config/crd/bases/%.yaml:
+# Is there any other file that needs to be a dependency here?
```

Comment:
This comment should be a PR review comment rather than a code comment. I'll remove it if we think there are no dependencies.

Reply:
I really think we should remove the parts we don't need from this project. I filed [HACBS-1521](https://issues.redhat.com/browse/HACBS-1521) for that which we can drop if we think there's value in keeping them.